portfolio/src/main/java/com/google/sps/data/Comment.java

```diff
@@ -0,0 +1,17 @@
+package com.google.sps.data;
+
+/** An item on the comment list. */
+public final class Comment {
```

Comment:
if this is a PoJo you might want to explore using lombok or protobuf

Reply:
Yes, it is a POJO and I'll learn about lombok and update this code.